### ISSUE TITLE
Fresh Install Error

### ISSUE THREAD

**Issue by tups** [antedebaas/DMARC-Reports#28]
Good morning,

I just installed the project on a server, checked that all packages were up to date with your recommendations.

I simply pull the project onto the server with GIT, and run update.sh.
Everything is going well except the Doctrine migration:

Voici l'erreur en console : 
```
run migrations
[notice] Migrating up to DoctrineMigrations\Version20240124174018
[notice] CREATE TABLE domains (id INT AUTO_INCREMENT NOT NULL, fqdn VARCHAR(255) NOT NULL, PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB
[error] Migration DoctrineMigrations\Version20230809184012 failed during Execution. Error: "An exception occurred while executing a query: SQLSTATE[HY000]: General error: 1 near "DEFAULT": syntax error"
[critical] Error thrown while running command "doctrine:migrations:migrate --no-interaction --query-time". Message: "An exception occurred while executing a query: SQLSTATE[HY000]: General error: 1 near "DEFAULT": syntax error"

In ExceptionConverter.php line 68:

  An exception occurred while executing a query: SQLSTATE[HY000]: General error: 1 near "DEFAULT": syntax error


In Exception.php line 28:

  SQLSTATE[HY000]: General error: 1 near "DEFAULT": syntax error


In Connection.php line 71:

  SQLSTATE[HY000]: General error: 1 near "DEFAULT": syntax error


doctrine:migrations:migrate [--write-sql [WRITE-SQL]] [--dry-run] [--query-time] [--allow-no-migration] [--all-or-nothing [ALL-OR-NOTHING]] [--configuration CONFIGURATION] [--em EM] [--conn CONN] [--] [<version>]

```

If I try to run the migration manually, php bin/console doctrine:migrations:migrate, I get the same error

Is there a recommendation for the SQLLite version?

Thank you

**Comment by antedebaas** [comment]
Unfortunately currently only MySQL/MariaDB is supported as SQLlite does not support all the requirements I need

**Comment by tups** [comment]
Ok, so if I understood correctly, I just need to change DATABASE_URL in the .env file, to choose MySQL for example

**Comment by tups** [comment]
I managed to start the project. It was an error on my part on the vhost
Good day

**Comment by antedebaas** [comment]
@tups i would advice to create an .env.local file.
everything in that fire will override values in the .env file.

the .env file gets updated from time to time, but ill never touch the .env.local

**Comment by tups** [comment]
THANKS :)
I'm not very familiar with Symfony.
In fact, I thought that the update had not worked and that `/setup` was sending me a 404 page because of that.
But after checking, I ended up finding an error in my Apache configuration.
And I ended up understanding that the `/setup` page creates the .env.local file

When you don't make any mistakes, it works very well :)

**Comment by antedebaas** [comment]
@tups
Sqlite is now implemented in v1.6.0
https://github.com/antedebaas/DMARC-SMTPTLS-Reports/releases/tag/v1.6.0
